src/utils/print-string.js

```diff
@@ -33,12 +33,10 @@ function printString(raw, options) {
   return makeString(
     rawContent,
     enclosingQuote,
-    !(
-      options.parser === "css" ||
-      options.parser === "less" ||
-      options.parser === "scss" ||
-      options.__embeddedInHtml
-    ),
+    // Until Prettier 3.3.3, this option was set to true for most parsers, with some exceptions like CSS.
+    // Since Prettier 3.3.4, it is set to false for all parsers.
+    // For more details, please see https://github.com/prettier/prettier/issues/16542#issuecomment-2282249280.
+    false,
```

Comment:
WDYT of removing this option completely, since it is always `false`?

This code can be deleted:
https://github.com/sosukesuzuki/prettier/blob/250847c2cc44c25f175a521d95daabc4ad804b9b/src/common/util.js#L328-L334

Reply:
makeString is a public API, so we cannot remove the option without major update

Reply:
Actually, we can, we exports many different versions of utilities in `public.js`, but this flag is not simply one line difference since it's used in a inner scope, so it's totally optional.